api/client/stats.go

```diff
@@ -97,12 +97,13 @@ func (s *containerStats) Display(w io.Writer) error {
 	if s.err != nil {
 		return s.err
 	}
-	fmt.Fprintf(w, "%s\t%.2f%%\t%s/%s\t%.2f%%\t%s/%s\n",
-		s.Name,
-		s.CPUPercentage,
-		units.BytesSize(s.Memory), units.BytesSize(s.MemoryLimit),
-		s.MemoryPercentage,
-		units.BytesSize(s.NetworkRx), units.BytesSize(s.NetworkTx))
+	io.WriteString(w,
```

Comment:
I haven't written a lot of Golang, but yes, it does seem appropriate. Let me update it. I blindly replaced all fmt.Fprint*